stock_prices/stock_prices.py

```diff
@@ -3,7 +3,15 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  maxProfit = 0
+  for index, price in enumerate(prices):
+    ##for all indexes after the current index, if the 2nd index value is larger, get the difference between the two
+      for index, nextPrice in enumerate(prices):
```

Comment:
You'll need to call this `index` `index2` instead if you want to use it, for example to check if `index2 > index`